### ISSUE TITLE
Prep 1.2 Release

### ISSUE THREAD

**Issue by richaber** [WebDevStudios/wp-search-with-algolia#45]
- [x] Bump Version to 1.2.0 in [Plugin Header](https://github.com/WebDevStudios/wp-search-with-algolia/blob/master/algolia.php#L7)
- [x] Bump [ALGOLIA_VERSION constant](https://github.com/WebDevStudios/wp-search-with-algolia/blob/master/algolia.php#L27) to 1.2.0
- [x] Bump Stable tag to 1.2.0 in [README.txt](https://github.com/WebDevStudios/wp-search-with-algolia/blob/master/README.txt#L8)
- [x] Add a basic, minimal [changelog](https://github.com/WebDevStudios/wp-search-with-algolia/blob/master/README.txt#L108) to README.txt
- [x] Add .gitattributes (prep for automated WPORG deploys)
- [x] Tag and release
- [x] Commit to WPORG SVN

**Comment by richaber** [comment]
Bump Version to 1.2.0 in Plugin Header addressed in commit bc0c583

Bump ALGOLIA_VERSION constant to 1.2.0 addressed in commit bc0c583

Bump Stable tag to 1.2.0 in README.txt addressed in commit b8793e5

Add a basic, minimal changelog to README.txt addressed in commit b8793e5

**Comment by richaber** [comment]
Our PHPCS checks are failing, because the codebase does not _currently_ conform to standards.

The following "info" report from PHPCS ignores `warnings` and only notes `errors` where we do not conform to standards:

```
phpcs --runtime-set ignore_warnings_on_exit 1 -d memory_limit=256M -p -s -n --standard=WebDevStudios --warning-severity=0 --report=info --extensions=php --ignore=*/index.php,*/includes/libraries/*,*/tests/*,*/vendor/*,*/node_modules/* --basepath=./ .
EEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE 34 / 34 (100%)

PHP CODE SNIFFER INFORMATION REPORT
----------------------------------------------------------------------
Adjacent assignments aligned: yes [72/72, 100%]

Assigning new by reference: no [41/41, 100%]

Class has doc comment: no [21/23, 91.3%]
	yes => 2 (8.7%)

Class opening brace placement: same line [24/24, 100%]

Closure opening brace placement: same line [7/7, 100%]

Constant name case: upper [9/9, 100%]

Control structure defined inline: no [234/234, 100%]

EOL char: \n [34/34, 100%]

File has doc comment: no [32/34, 94.12%]
	yes => 2 (5.88%)

Function has doc comment: yes [205/257, 79.77%]
	no => 52 (20.23%)

Function opening brace placement: same line [246/246, 100%]

Line indent: tabs [4063/4085, 99.46%]
	spaces => 22 (0.54%)

Multiple statements on same line: no [1157/1157, 100%]

Newline at EOF: yes [34/34, 100%]

Object instantiation with parenthesis: yes [41/41, 100%]

PHP closing tag at end of PHP-only file: no [23/23, 100%]

PHP constant case: lower [110/110, 100%]

PHP keyword case: lower [1558/1558, 100%]

PHP short open tag used: no [624/624, 100%]

PHP type case: lower [132/132, 100%]

Space after operator: 1 [887/888, 99.89%]
	0 => 1 (0.11%)

Space before operator: 1 [271/275, 98.55%]
	newline => 3 (1.09%)
	2 => 1 (0.36%)

Space between classname and parenthesis: 0 [41/41, 100%]

Spacing after cast statement: 1 [118/118, 100%]

Spacing after object operator: 0 [712/712, 100%]

Spacing after string concat: 1 [164/167, 98.2%]
	newline => 3 (1.8%)

Spacing before object operator: 0 [712/712, 100%]

Spacing before string concat: 1 [165/167, 98.8%]
	newline => 2 (1.2%)

Use of ELSE IF or ELSEIF: elseif [5/5, 100%]

Using byte order mark: no [6/6, 100%]

----------------------------------------------------------------------
```

Most notably there are a lot of files/methods/functions missing docblocks, and a few spacing issues.

There are significantly more warnings than errors. All of these issues can be addressed later, as most were inherited.

**Comment by richaber** [comment]
Add .gitattributes (prep for automated WPORG deploys) addressed in commit 189d546

**Comment by richaber** [comment]
Version [1.2.0](https://github.com/WebDevStudios/wp-search-with-algolia/releases/tag/1.2.0) tagged and released

**Comment by richaber** [comment]
WP Search with Algolia version 1.2.0 tagged and released.

- Available in this project's releases under the [1.2.0](https://github.com/WebDevStudios/wp-search-with-algolia/releases/tag/1.2.0) tag.
- Available for download at [WordPress.org](https://wordpress.org/plugins/wp-search-with-algolia/).
